.gitignore

```diff
@@ -28,3 +29,15 @@ package-lock.json
 # IDE
 .idea/
 .viminfo
+
+# Keys for webdriver
+config.json
+
+# Temp files
+app/public/summary
+dip_trades/dip_log.txt
+dip_trades/yarn.lock
+failure.html
+screenshot.png
+trading_report/offers.csv
```

Comment:
Will the daily workflow still update the site without these?

Reply:
Yes. The daily report update moves it to somewhere else that it is tracked by git